packages/core/entity/FileExplorerURL/index.ts

```diff
@@ -157,4 +157,79 @@ export default class FileExplorerURL {
             sortColumn,
         };
     }
+
+    public static convertToPython(urlComponents: Partial<FileExplorerURLComponents>) {
+        const collectionString = this.convertCollectionToPython(urlComponents?.collection);
+
+        const groupByQueryString =
+            urlComponents.hierarchy
+                ?.map((annotation) => this.convertGroupByToPython(annotation))
+                .join("") || "";
+
+        // Group filters by name and use OR to concatenate same filter values
+        const filterGroups = new Map();
+        urlComponents.filters?.forEach((filter) => {
+            const pythonQueryString = filterGroups.get(filter.name);
+            if (!pythonQueryString) {
+                filterGroups.set(filter.name, this.convertFilterToPython(filter));
+            } else {
+                filterGroups.set(
+                    filter.name,
+                    pythonQueryString.concat(` | ${this.convertFilterToPython(filter)}`)
+                );
+            }
+        });
+
+        // Chain the filters together
+        let filterQueryString = "";
+        filterGroups.forEach((value) => {
+            filterQueryString = filterQueryString.concat(`.query('${value}')`);
+        });
+
+        const sortQueryString = urlComponents.sortColumn
+            ? this.convertSortToPython(urlComponents.sortColumn)
+            : "";
+        // const fuzzy = [] // TO DO: support fuzzy filtering
+
+        const hasQueryElements = groupByQueryString || filterQueryString || sortQueryString;
+        const imports = "import pandas\n";
```

Comment:
FWIW I think most people using python might be more familiar with `import pandas as pd`

Reply:
updated in [4c46839](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/98/commits/4c46839ed84944cd4653718cf96ba1c93cd5d293)